build.gradle

```diff
@@ -27,6 +16,16 @@
 	configurations.classpath.resolutionStrategy.cacheDynamicVersionsFor 0, 'minutes'
 }
 
+plugins {
+	alias libs.plugins.license
+	alias libs.plugins.nebulaRelease
+	alias libs.plugins.nebulaPublishing
+	alias libs.plugins.nebulaProject
+	alias libs.plugins.noHttp
+	alias libs.plugins.nexusPublish
+	alias libs.plugins.javaformat
```

Comment:
Declaring plugins in the `plugins` block by default applies them. When a dependency is declared in the `buildscript`, the plugin isn't applied unless `apply plugin:` is run for a project. Some of these plugins shouldn't be applied to every project in other repos. There's only one project (module) in this repo, so it wasn't an issue. I'll update anyway so it's easier to apply these changes to other repos.

Reply:
I would move this back to the old way of applying plugins:
```
buildscript {
    dependencies {
        classpath '...'
    }
}
```

The reason is it is more explicit and you can always declare `buildscript` dependencies but there are places where you cannot do `plugins {}` (I forgot where).

Reply:
It's a fair point that we could consider the change from buildscript to plugins block separately from the TOML migration so we aren't changing more than one thing at a time. Eventually, I'd like to move in the direction of getting rid of the `buildscript` block entirely, if we could. It's, as far as I understand, the [legacy way of doing plugins](https://docs.gradle.org/current/userguide/plugins.html#sec:old_plugin_application). I don't feel strongly about it, but this did feel cleaner.